extras/fixture/src/unity_fixture.c

```diff
@@ -178,7 +186,7 @@ void UnityMalloc_MakeMallocFailAfterCount(int countdown)
 
 #ifdef UNITY_EXCLUDE_STDLIB_MALLOC
 static unsigned char unity_heap[UNITY_INTERNAL_HEAP_SIZE_BYTES];
-static size_t heap_index;
+static size_t        heap_index;
```

Comment:
again the alignment here I doubt it is what we expect/have elsewhwere?

Reply:
See above.